src/bundle/Command/MigrateNamespacesCommand.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\FieldTypeRichText\Command;
+
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\FieldTypeRichText\Persistence\Legacy\ContentModelGateway as Gateway;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class MigrateNamespacesCommand extends MultiprocessComand
+{
+    private Gateway $gateway;
+
+    private ?int $cursorStart;
+
+    private ?int $cursorStop;
+
+    public function __construct(
+        PermissionResolver $permissionResolver,
+        UserService $userService,
+        Gateway $gateway,
+    ) {
+        parent::__construct('ibexa:migrate:richtext-namespaces', $permissionResolver, $userService);
+        $this->gateway = $gateway;
+    }
+
+    public function configure(): void
+    {
+        parent::configure();
+
+        $this->addOption(
+            'cursor-start',
+            null,
+            InputOption::VALUE_REQUIRED,
+            'Internal option - only used for subprocesses',
+        )
+            ->addOption(
+                'cursor-stop',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Internal option - only used for subprocesses',
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->cursorStart = $input->getOption('cursor-start') !== null ? (int) $input->getOption('cursor-start') : null;
+        $this->cursorStop = $input->getOption('cursor-stop') !== null ? (int) $input->getOption('cursor-stop') : null;
+
+        // Check that both --cursor-start and cursor-start are set, or neither
+        if (($this->cursorStart === null) xor ($this->cursorStop === null)) {
+            throw new RuntimeException('The options --cursor-start and -cursor-stop are only for internal use !');
+        }
+
+        parent::execute($input, $output);
+
+        return self::SUCCESS;
+    }
+
+    protected function getObjectCount(): int
+    {
+        return $this->gateway->countRichtextAttributes();
+    }
+
+    protected function iterate(): void
+    {
+        $limit = $this->getIterationCount();
+        $cursor = [
+            'start' => -1,
+            'stop' => null,
+        ];
+
+        $contentAttributeIDs = $this->gateway->getContentObjectAttributeIds($cursor['start'], $limit);
```

Comment:
Since `contentobject_attribute.id` is not unique, I found no better way than to fetch every single `id` in the mother process as I wanted to avoid using `offset` in the SQLs later on when iterating over the dataset